OnlineShopUI/online-shop-ui/src/components/Login.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Modal from '@mui/material/Modal';
+import CloseIcon from '@mui/icons-material/Close';
+import './Login.scss';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import User from '../img/icon-user.png';
+
+export default function Login() {
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
```

Comment:
We don't understand what we handle here, it's better to name 'handleLogInModalOpen'